### PR TITLE
Hub: Modifies the rating icons on detail page

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

Initial rating icons:

image

Modified rating icons :

image

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: hub/ui-new/src/components/rating/index.css [hunk 1]
```diff
@@ -24,36 +24,41 @@
...
   border-style: none;
-  margin-left: 4em;
+  margin-left: 5.9em;
 }
 
 .rate-area:not(:checked)>input {
   position: absolute;
   top: -9999px;
-  clip: rect(0, 0, 0, 0);
+  /*clip: rect(0, 0, 0, 0);*/
```
Comment (sthaha) [inline hub/ui-new/src/components/rating/index.css:33]: could we remove this if unused?

File: hub/ui-new/src/components/rating/index.css [hunk 2]
```diff
@@ -24,36 +24,41 @@
...
 }
 
 .rate-area:not(:checked)>label {
   float: right;
   width: 1em;
-  padding: 0 .1em;
+  /*padding: 0 .1em;*/
```
Comment (sthaha) [inline hub/ui-new/src/components/rating/index.css:39]: same...

File: hub/ui-new/src/components/rating/index.css [hunk 3]
```diff
@@ -24,36 +24,41 @@
...
   overflow: hidden;
   white-space: nowrap;
   cursor: pointer;
-  font-size: 250%;
+  font-size: 170%;
   line-height: 1.2;
   color: lightgrey;
-  text-shadow: 1px 1px #484848;
+  /*text-shadow: 1px 1px #484848;*/
```
Comment (sthaha) [inline hub/ui-new/src/components/rating/index.css:46]: 🤔

File: hub/ui-new/src/components/rating/Rating.tsx [hunk 4]
```diff
@@ -141,44 +142,62 @@ const Rating: React.FC = (props: any) => {
   if (props.isAuthenticated === true) {
     login = <form onClick={updateRating}>
       <ul className="rate-area" >
-        <input type="radio" id="5-star"
-          name="rating" value="5" checked={fivechecked} />
+        <input type="radio" id="5-star" name="rating"
```
Comment (sthaha) [inline hub/ui-new/src/components/rating/Rating.tsx:145]: could we use the same formatting as the block below? 

```react
<input ... />
<label ... > <StarIcon /> </label>
```

Comment (PuneetPunamiya) [reply hub/ui-new/src/components/rating/Rating.tsx:145]: It was giving an `eslint` error of limited chars in a line

**Pull request merged**